Cargo.toml

```diff
@@ -1,6 +1,39 @@
-[workspace]
-members = ["crates/miden-multisig-client"]
-resolver = "2"
+[workspace.package]
+edition = "2024"
+publish = false
+version = "0.0.1-pre"
 
 [workspace.lints.rust]
 missing_docs = "warn"
+unsafe_code  = "forbid"
+
+[workspace]
+members = [
+  "bin/coordinator/domain",
+  "bin/coordinator/engine",
+  "bin/coordinator/server",
+  "bin/coordinator/store",
+  "crates/*",
+]
+resolver = "3"
+
+[workspace.dependencies]
+miden-client                      = "0.11.9"
```

Comment:
```suggestion
miden-client                      = "0.11"
```

Reply:
The workflow would ideally be to run `cargo update -p <package name>`, so that we always use the version checked into `Cargo.lock`. This is the flow we follow in other repositories, too.
This way, the developer of the crate decides when to upgrade and to which version. Cargo will by default override patch versions because they're semver compatible.